velox/docs/monthly-updates/june-2022.rst

```diff
@@ -0,0 +1,67 @@
+****************
+June 2022 Update
+****************
+
+Documentation
+=============
+
+* Add documentation for :ref:`complex types writers<outputs-write>`.
+
+Core Library
+============
+
+* Add support for INTERVAL DAY TO SECOND Presto type.
+* Allow cast between DATE type and TIMESTAMP type.
+* Allow cast from JSON type to scalar types, Array type, and Map type.
+* Add :ref:`GroupIdNode<group-id-node>` and GroupId operator to support aggregations over grouping sets.
+* Add support for DECIMAL type signature and type calculation using flex/bison.
```

Comment:
Add support for function signatures with DECIMAL input and return types using flex and bizon to evaluate formulas for calculating the return precision and scale based on input precisions and scales.